games-strategy/0ad_data/0ad_data-0.0.22.recipe

```diff
@@ -0,0 +1,28 @@
+SUMMARY="O A.D. - Game Data"
+DESCRIPTION="Data for the game 0 A.D."
+HOMEPAGE="https://play0ad.com/"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-data.tar.gz"
+CHECKSUM_SHA256="b090decbf99a978d92642eb7d813b01da3c2c815ce3ffcd8795727e2df998f30"
+SOURCE_DIR="0ad-$portVersion-alpha"
+
+ARCHITECTURES="any"
+
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	0ad_data = $portVersion
+"
+
+BUILD()
+{
+	true
+}
+
+INSTALL()
+{
+	mkdir -p $dataDir
+	cp -R * $appsDir/0ad
```

Comment:
What you need in **0ad_data**'s `INSTALL()` is domething like:
```
INSTALL()
{
	install -d $dataDir/0ad
	install -t $dataDir/0ad *
}
```
Then, in **0ad**'s `BUILD()` you need to configure it, stating that the data from 0ad_data will be available in `$portPackageLinksDir/0ad_data/data/0ad/`.

Reminder: everything in there is read-only.

To have an idea of what $portPackageLinksDir is, you can search for portPackageLinksDir in the file called wrapper-script that you can find in the work directory: work-0.0.22.

To see how an example of how a package can use portPackageLinksDir to access files from other packages, you can check  subversion-1.6.18.recipe (in dev-vcs/subversion/.)

BTW, during the build, the data from 0ad_data will be available thanks to the `0ad_data == $portVersion` line in 0ad's BUILD_REQUIRES.